projects/vscode-atl-render/syntaxes/atl-render.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+  "scopeName": "atl.render",
+  "injectionSelector": "L:source.ts -comment -(string -text.html)",
+  "name": "atl.render",
+  "patterns": [
+    {
+      "include": "#renderMethod"
+    }
+  ],
+  "repository": {
+    "renderMethod": {
+      "name": "renderMethod",
+      "begin": "(?x)(\\b(?:\\w+\\.)*(?:render)\\s*)(\\()",
+      "beginCaptures": {
+        "1": {
+          "name": "entity.name.function.ts"
+        },
+        "2": {
+          "name": "meta.brace.round.ts"
+        }
+      },
+      "end": "(\\))",
+      "endCaptures": {
+        "0": {
+          "name": "meta.brace.round.ts"
+        }
+      },
+      "patterns": [
+        {
+          "include": "#renderTemplate"
+        }
+      ]
+    },
+    "renderTemplate": {
+      "contentName": "text.html",
+      "begin": "\\G\\s*([`|'|\"])",
```

Comment:
Maybe it disallows line breaks between `render(` and the arguments.

> Note that the regular expressions are matched against only a single line of the document at a time. That means it is not possible to use a pattern that matches multiple lines. The reason for this is technical: being able to restart the parser at an arbitrary line and having to re-parse only the minimal number of lines affected by an edit. In most situations it is possible to use the begin/end model to overcome this limitation.

https://macromates.com/manual/en/language_grammars

How about by just `([`|'|\"])`?

BTW, `inline-template.ng` also disallows line breaks. 
![image](https://user-images.githubusercontent.com/1529180/121793618-d8f4f100-cc3b-11eb-9e67-b68b43494082.png)

Reply:
\\G is "matching start position", so maybe it means \\G is the same line of render(.